views/pages/EditChartPage.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react'
+import { useParams, useOutletContext } from 'react-router-dom'
+
+import ChartForm from '../forms/ChartForm'
+import { editChart, getChart } from '../fe-utils/fetchUtil'
+import { routes } from '../routes/routes'
+
+const EditChartPage = () => {
+  const { chart_id } = useParams()
+  const [chart, setChart] = useState()
+  const { user, classes, navigate } = useOutletContext()
+
+  const handleSubmit = async (e, formValues) => {
+    e.preventDefault()
+    const { data } = await editChart(chart_id, formValues)
+
+    if (data.modifiedCount === 1) navigate(routes.viewChart(chart_id))
```

Comment:
this shouldn't return the modified count, but the updated chart.

Reply:
With the changes to the api it will be addressed during the charts refactor